examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -226,15 +231,13 @@ def loss_fxn(ff_params, mol_a, mol_b, core, label_ddG):
     ordered_params = forcefield.get_ordered_params()
     ordered_handles = forcefield.get_ordered_handles()
 
-    handle_types_being_optimized = [AM1CCCHandler, LennardJonesHandler]
```

Comment:
Was duplicated by forces_to_refit and seemed clearer to have it at the top near the comment about parameterizing it